### PR TITLE
chore(release): prepare for release

### PR DESCRIPTION
**Pull request by russellwheatley**
## Description

See title

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database](https://github.com/flutter/flutter/issues). Indicate, which of these issues are resolved or fixed by this PR. Note that you'll have to prefix the issue numbers with flutter/flutter#.*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [ ] I read and followed the [Flutter Style Guide].
- [ ] I signed the [CLA].
- [ ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [ ] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/firebase/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by russellwheatley** [comment]
A few manual edits I had to make:

1. Manually edit crashlytics so versions have a patch update and removed log for build ID commit that was reverted.
2. Updated in-app-messaging & app-check so they showed an internal deprecated API refactor.

File: packages/firebase_messaging/firebase_messaging/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 13.0.0
+
+> Note: This release has breaking changes.
+
+ - **BREAKING** **FEAT**: android 13 notifications permission request ([#9348](https://github.com/firebase/flutterfire/issues/9348)). ([43b3b06b](https://github.com/firebase/flutterfire/commit/43b3b06b64739658f79c994110654f5a56abca05))
```
Comment (Lyokone) [inline packages/firebase_messaging/firebase_messaging/CHANGELOG.md:5]: I think we should mention why is this breaking with more information like we mentioned here: https://github.com/firebase/flutterfire/pull/9348/files#r946825531

**Review by Lyokone — CHANGES_REQUESTED** [review]
Some edit needed for breaking change

File: packages/firebase_messaging/firebase_messaging/CHANGELOG.md (hunk 1 above)
Comment (russellwheatley) [reply packages/firebase_messaging/firebase_messaging/CHANGELOG.md:5]: Ah, I added that to the description when I squashed it which I thought added to the Changelog 🤔 . Will update 👍

Comment (Lyokone) [reply packages/firebase_messaging/firebase_messaging/CHANGELOG.md:5]: Huuum do we have the right GitHub setting 🤔

**Review by Lyokone — APPROVED** [review]

File: packages/firebase_messaging/firebase_messaging/CHANGELOG.md (hunk 1 above)
Comment (russellwheatley) [reply packages/firebase_messaging/firebase_messaging/CHANGELOG.md:5]: Either that or melos 🤷 . Not sure without invesitgating.

Comment (Salakar) [reply packages/firebase_messaging/firebase_messaging/CHANGELOG.md:5]: > Ah, I added that to the description when I squashed it which I thought added to the Changelog 🤔 . Will update 👍

Melos only adds it if you do it in the following format in the commit body (conventional commits format):

```

BREAKING: Some text here about why it's breaking.

```

**Review by Salakar — APPROVED** [review]

**Pull request merged**